R-package/R/utils.R

```diff
@@ -217,3 +217,27 @@ lgb.check.wrapper_param <- function(main_param_name, params, alternative_kwarg_v
   params[[main_param_name]] <- alternative_kwarg_value
   return(params)
 }
+
+#' @importFrom parallel detectCores
+lgb.get.default.num.threads <- function() {
+  # nolint
+  if (require("RhpcBLASctl", quietly = TRUE)) {
```

Comment:
```suggestion
  if (require("RhpcBLASctl", quietly = TRUE)) {. # nolint
```

The `# nolint` needs to be on this line for `{lintr}` to respect it.